File: 1.programming/c/snippets/operators_math.c
#include <stdio.h>
int main() {

  int num, num2;

  printf("Enter a number: ");
  scanf("%i", &num);
  printf("Enter another number: ");
  scanf("%i", &num2);

  printf("%i + %i = %i\n", num, num2, num + num2);
  printf("%i - %i = %i\n", num, num2, num - num2);
  printf("%i * %i = %i\n", num, num2, num * num2);
  printf("%i / %i = %i\n", num, num2, num / num2);
  return 0;
}

File: 1.programming/go/snippets/operators_math.go
package main

import "fmt"

func main() {
	var num int
	var num2 int
	fmt.Println("Please enter a number: ")
	fmt.Scanln(&num)
	fmt.Println("Please enter another number: ")
	fmt.Scanln(&num2)

	fmt.Println(num, " + ", num2, " = ", num+num2)
	fmt.Println(num, " - ", num2, " = ", num-num2)
	fmt.Println(num, " * ", num2, " = ", num*num2)
	fmt.Println(num, " / ", num2, " = ", num/num2)
}
